index.html

```diff
@@ -29,6 +29,7 @@
     </div>
 
   <section id="workTimeSection" class="middle container-rect">
+    <span class="input-group-addon">Start at </span>
```

Comment:
This span is in the wrong element. Should be placed in the clockpicker div below.

Reply:
Sorry! I corrected that.